addons/account/report/account_invoice_report_view.xml

```diff
@@ -41,6 +41,7 @@
                 <field name="company_id"  groups="base.group_multi_company"/>
                 <field name="price_average" optional="hide" sum="Total"/>
                 <field name="quantity" optional="hide" sum="Total"/>
+                <field name="price_subtotal_currency" optional="show" sum="Total"/>
```

Comment:
not sure but i think optional show is the basic behavior no ? Again we could choose to be consistent with the file :thinking:

Reply:
Indeed, `optional="show"` is the default behaviour. I will keep it as it's to maintain consistency.